File: PHP PowerPoint Examples/SlidesApiDemo/addSlide.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "test.pptx";
$password = null;
$folder = "input";
$storage = null;
$position = 0;
$layoutAlias = "TitleOnly"; // TwoTextAndTwoObjects/ Title/ TextAndObject/ Blank/ VerticalTitleAndText/ PictureAndCaption/ TitleAndObject/ TwoObjects
$apiInstance->addSlide($name, $position, $password, $folder, $storage, $layoutAlias);

File: Python PowerPoint Examples/SlidesApiDemo/addSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "TitleOnly.pptx"
position = 0
password = ""
storage = ""
inputFolder = "input"
#supports layoutAlias: TitleOnly/Title/123/null/Blank/TwoTextAndTwoObjects/SectionHeader/TwoObjects/TitleObjectAndCaption/TitleAndObject/PictureAndCaption/VerticalTitleAndText/
layoutAlias = "TitleOnly"
result =api.add_slide(fileName, position=position, password=password, storage=storage, folder=inputFolder,layout_alias=layoutAlias)